### PR TITLE
Issue1171 storage spring cleaning

### PR DESCRIPTION
**Pull request by LauraMaier**
This closes issue #1171

### CODE REVIEW

File: AixLib/Fluid/Storage/StorageDetailed.mo [hunk 1]
```diff
@@ -36,7 +36,8 @@ model BufferStorage
 
   parameter SI.Temperature TStart=298.15 "Start Temperature of fluid" annotation (Dialog(tab="Initialization", group="Storage specific"));
 
-  replaceable parameter DataBase.Storage.BufferStorageBaseDataDefinition data constrainedby DataBase.Storage.BufferStorageBaseDataDefinition "Data record for Storage"
+  replaceable parameter DataBase.Storage.BufferStorageBaseDataDefinition data constrainedby
+    DataBase.Storage.BufferStorageBaseDataDefinition                                                                                         "Data record for Storage"
```
Comment (FWuellhorst) [inline AixLib/Fluid/Storage/StorageDetailed.mo:40]: Should this record then be called "DetailedStorageBaseDataDefinition"?

File: AixLib/Fluid/Storage/StorageDetailed.mo [hunk 2]
```diff
@@ -166,6 +170,12 @@ model BufferStorage
       "Heat transfer model" annotation (Placement(transformation(extent={{-34,0},
             {-14,20}}, rotation=0)));
 
+  Modelica.Fluid.Interfaces.FluidPort_b portHC1Out(redeclare final package
+      Medium =
+        MediumHC1) if useHeatingCoil1
+    "Fluid connector b (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{-90,2},{-70,22}}),
+        iconTransformation(extent={{-88,20},{-74,32}})));
```
Comment (FWuellhorst) [inline AixLib/Fluid/Storage/StorageDetailed.mo:178]: Why move this HC1 port to the code part above HC1 comment?

File: AixLib/Resources/Scripts/ConvertAixLib_from_1.0.2_to_1.0.3.mos [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿clear
+
+convertClear();
+
+// Rename Buffer and standard storage
+convertClass("AixLib.Fluid.Storage.BufferStorage", "AixLib.Fluid.Storage.StorageDetailed");
```
Comment (FWuellhorst) [inline AixLib/Resources/Scripts/ConvertAixLib_from_1.0.2_to_1.0.3.mos:6]: If you accept the name changes above, add the conversion here.

File: AixLib/Systems/TABS/BaseClasses/TabsBus.mo [hunk 4]
```diff
@@ -10,11 +10,14 @@ expandable connector TabsBus "Data bus for tabs"
          coordinateSystem(preserveAspectRatio=false)),
     Diagram(graphics,
             coordinateSystem(preserveAspectRatio=false)),
-    Documentation(info="<html>
-<p>Definition of a bus connector for the Tabs Module.</p>
+    Documentation(info="<html><p>
+  Definition of a bus connector for the Tabs Module.
+</p>
```
Comment (FWuellhorst) [inline AixLib/Systems/TABS/BaseClasses/TabsBus.mo:15]: Why did you change all these TABS models?

**Review by FWuellhorst — CHANGES_REQUESTED** [review]
Nice work! Remove TABS changes and check comments.

File: AixLib/Fluid/Storage/StorageDetailed.mo (hunk 1 above)
Comment (LauraMaier) [reply AixLib/Fluid/Storage/StorageDetailed.mo:40]: Good idea, changed it in the latest push

File: AixLib/Resources/Scripts/ConvertAixLib_from_1.0.2_to_1.0.3.mos (hunk 3 above)
Comment (LauraMaier) [reply AixLib/Resources/Scripts/ConvertAixLib_from_1.0.2_to_1.0.3.mos:6]: Added conversion in latest push

File: AixLib/Systems/TABS/BaseClasses/TabsBus.mo (hunk 4 above)
Comment (LauraMaier) [reply AixLib/Systems/TABS/BaseClasses/TabsBus.mo:15]: Automatic CI merge

Comment (LauraMaier) [reply AixLib/Systems/TABS/BaseClasses/TabsBus.mo:15]: Automatic CI merge...

File: AixLib/Fluid/Storage/StorageDetailed.mo (hunk 2 above)
Comment (LauraMaier) [reply AixLib/Fluid/Storage/StorageDetailed.mo]: Good question, rereplaced it back to old position

Comment (LauraMaier) [reply AixLib/Fluid/Storage/StorageDetailed.mo]: good question - rechanged this

**Review by FWuellhorst — APPROVED** [review]
Nice work. Just the CI now ;)

**Comment by FWuellhorst** [comment]
@LauraMaier : Could you update the branch to MSL 4.0? Then I will add my review.

**Comment by FWuellhorst** [comment]
@LauraMaier : Even though the plots dont show, here is the error:
```
*** Warning: Unspecified initial conditions in 'RunScript("Resources/Scripts/Dymola/Fluid/Storage/Examples/StorageSolarCollector.mos");'.
*** Warning: Unspecified initial conditions in 'RunScript("Resources/Scripts/Dymola/Fluid/Storage/Examples/StorageBoiler.mos");'.
```

**Comment by LauraMaier** [comment]
@FWuellhorst and @DaJansenGit I tried to fix the problem with the initial conditions which led to a CI fail.
However, the pipeline does not do the full check and simply tells me that checks have passed. What do I have to do to actually check if the PR is ready to be merged now?

**Comment by LauraMaier** [comment]
@FWuellhorst Will you be able to re-review in October? :) That would be great ! :)

**Review by FWuellhorst — APPROVED** [review]
Thanks for this cleaning up! Looks good!

**Review by FWuellhorst — APPROVED** [review]
Thanks for the feature, I will merge this now

**Pull request merged**